### PR TITLE
DOC-3566 multiple pgsql databases topic

### PR DESCRIPTION
**Pull request by aimurphy**
## Major changes

New topic: [Run multiple PostgreSQL instances in Background steps](https://64d66943c55b450a85289f25--harness-developer.netlify.app/docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres)

## Minor changes

Updated shell, entry point, and command for [Background step settings](https://64d66943c55b450a85289f25--harness-developer.netlify.app/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings)

Add more links to examples on [Dependency management strategies](https://64d66943c55b450a85289f25--harness-developer.netlify.app/docs/continuous-integration/use-ci/manage-dependencies/dependency-mgmt-strategies)

### CODE REVIEW

**Review by raghavharness — APPROVED** [review]
Looks good, thanks for adding it for both k8 and hosted.

File: docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+* Enter a **Name** and note the **Id**. You need the **Id** to call the service in later steps in this stage.
+* For **Container Registry** and **Image**, select a Docker connector and specify a [PostgreSQL Docker image](https://hub.docker.com/_/postgres).
+* Add the following **Entry Point** arguments, and replace `PORT_NUMBER` with the desired port binding for this database instance, such as `5433`.
+   * `docker-entrypoint.sh`
+   * `-p PORT_NUMBER`
+* Add the following **Environment Variables**:
+   * `POSTGRES_USER`: Postgres database user name.
+   * `POSTGRES_DB`: Name for the database.
+   * `POSTGRES_PASSWORD`: Password for the specified user/database. You can use an expression to [reference a Harness text secret](/docs/platform/secrets/add-use-text-secrets/).
+* Configure [other settings](./background-step-settings), if needed.
```
Comment (douglas-j-bothwell) [inline docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md:61]: Another 404

File: docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+
+* Configure [other settings](./background-step-settings), if needed.
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+3. Select **Apply Changes** to save the step.
+4. Run your pipeline to test your PostgreSQL services. You can monitor and review build logs on the [Build details page](../viewing-builds). Once you've confirmed that the services run as expected, you can configure the rest of your pipeline. Note that **Background** steps don't persist across stages.
```
Comment (douglas-j-bothwell) [inline docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md:244]: Another 404.

File: docs/continuous-integration/use-ci/manage-dependencies/background-step-settings.md [hunk 3]
```diff
@@ -78,17 +83,19 @@ The stage's build infrastructure determines whether these fields are required or
...
 
-For **Shell**, select the shell script type for the arguments and commands defined in **Entry Point** and **Command**. Options include: **Bash**, **PowerShell**, **Pwsh**, **Sh**, and **Python**. If the step includes commands that aren't supported for the selected shell type, the build fails. Required binaries must be available on the build infrastructure or the specified image, as described in [Container Registry and Image](#container-registry-and-image).
+### Shell
 
-For **Entry Point** supply a list of arguments in `exec` format. **Entry Point** arguments override the image `ENTRYPOINT` and any commands in the **Command** field. Enter each argument separately.
+Select the shell type for the commands defined in **Entry Point** or **Command**. Options include: **Bash**, **PowerShell**, **Pwsh** (PowerShell Core), **Sh**, and **Python**. If the step includes commands that aren't supported for the selected shell type, the build fails. Required binaries must be available on the build infrastructure or the specified image, as described in [Container Registry and Image](#container-registry-and-image).
```
Comment (douglas-j-bothwell) [inline docs/continuous-integration/use-ci/manage-dependencies/background-step-settings.md:88]:
```suggestion
Select the shell type for the commands defined in **Entry Point** or **Command**. Options include: **Bash**, **PowerShell**, **Pwsh** (PowerShell Core), **Sh**, and **Python**. If the step includes commands that aren't supported for the selected shell type, the build fails. Required binaries must be available on the build infrastructure or the specified image. For more information, go to [Container Registry and Image](#container-registry-and-image).
```

**Review by douglas-j-bothwell — CHANGES_REQUESTED** [review]
Quite a few 404 links in the postGRES topic. Otherwise looks good.

File: docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+
+   For example:
+
+   ```
+   sleep 15
+   psql -U postgres -d test -h Background_1 -p 5433
+   psql -U postgres -d test1 -h Background_2 -p 5434
+   ```
+
+* Configure [other settings](./background-step-settings.md), if needed.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md:210]:
```suggestion
* Configure [other settings](../run-ci-scripts/run-step-settings.md#settings), if needed.
```

File: docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md [hunk 5]
```diff
@@ -0,0 +1,470 @@
...
+
+   For example:
+
+   ```
+   sleep 15
+   psql -U postgres -d test -h localhost -p 5433
+   psql -U postgres -d test1 -h localhost -p 5434
+   ```
+
+* Configure [other settings](./background-step-settings.md), if needed.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/manage-dependencies/multiple-postgres.md:236]:
```suggestion
* Configure [other settings](../run-ci-scripts/run-step-settings.md#settings), if needed.
```

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**